mixer/doc/deploy-mixer-with-custom-adapter.md

```diff
@@ -0,0 +1,139 @@
+# Deploying Mixer with custom adapters
+
+Congratulations! You've managed to build a complete Mixer adapter and have validated
+that it works via local testing.
+
+Follow these steps to deploy a Mixer with your custom adapter to your service
+mesh for use.
+
+1. Build a docker image of Mixer with your custom adapter.
+
+   Execute the following command:
+
```

Comment:
Consider clarifying which directory to start these steps.

Reply:
Added a NOTE at top.